source/groupmeeting/talks.rst

```diff
@@ -7,6 +7,18 @@ Some invited talks in MIGG, with some PPTs in MIGG Google Drive (``groupmeeting.
 2021
 ----
 
+Seismic Observayion and Imaging Using Dense Seismic Arrays with Nodes and Distributed Acoustic Sensing (DAS)
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+- 2021/03/18 by Feng Cheng
+
+- Abstract: 
+Recently, dense seismic deployments, with hundreds to thousands of short-period nodal instruments or distributed acoustic sensing (DAS) systems, have been widely used in seismological studies. These dense arrays have very close station spacings ranging from several meters to hundreds of meters to record well-sampled and unaliased wavefields in local or regional settings. Data acquired by such dense systems promote the development of new array-based seismic imaging methods and greatly improve our understanding of fine-scale subsurface properties, microseismic activities and earthquake rupture processes. During the presentation, Dr. Cheng will briefly present his works on dense seismic array observation and high-resolution seismic imaging, including geology structure delineation with 1D linear dense array, geothermal reservoir exploration using 2D dense array, hidden geothermal systems characterization using DAS array. Discussions related to dense array deployment, observation, imaging will be encouraged during and after the presentation.
```

Comment:
Need to have an indentation same as `Abstract`. We use reStrcuturedText to write the documentation. If you want to know more, please check the following two tutorials:
- https://docs.generic-mapping-tools.org/latest/rst-cheatsheet.html (in English)
- https://seismo-learn.org/contributing/restructuredtext/ (in Chinese)
```suggestion
  Recently, dense seismic deployments, with hundreds to thousands of short-period nodal instruments or distributed acoustic sensing (DAS) systems, have been widely used in seismological studies. These dense arrays have very close station spacings ranging from several meters to hundreds of meters to record well-sampled and unaliased wavefields in local or regional settings. Data acquired by such dense systems promote the development of new array-based seismic imaging methods and greatly improve our understanding of fine-scale subsurface properties, microseismic activities and earthquake rupture processes. During the presentation, Dr. Cheng will briefly present his works on dense seismic array observation and high-resolution seismic imaging, including geology structure delineation with 1D linear dense array, geothermal reservoir exploration using 2D dense array, hidden geothermal systems characterization using DAS array. Discussions related to dense array deployment, observation, imaging will be encouraged during and after the presentation.
```